src/plugins/dataflow/components/use-link-table-dialog-dataflow.tsx

```diff
@@ -54,21 +57,25 @@ interface IProps {
   handleRequestTableLink: ((tableId: string) => void) | undefined;
   handleRequestTableUnlink: ((tableId: string) => void) | undefined;
 }
-export const useLinkTableDialog = ({ tableTiles, model, handleRequestTableLink, handleRequestTableUnlink }: IProps) => {
+
+export const useLinkTableDialogDataFlow = ({ tableTiles, model, handleRequestTableLink,
+  handleRequestTableUnlink }: IProps) => {
+
   const [selectValue, setSelectValue] = useState("");
+  const content = model.content as DataflowContentModelType;
+
   const handleClick = () => {
-    const _content = model.content as DataflowContentModelType;
+    // const _content = model.content as DataflowContentModelType;
```

Comment:
This code should be removed.

Reply:
Actually we want to remove all of `line 68`, since we've defined another variable `content` right above it on `65`. @tealefristoe pointed out `_content` isnt necessary. And handleClick() should have scope of `content`